src/rpc/rpcquorums.cpp

```diff
@@ -11,18 +11,18 @@
 void quorum_dkgstatus_help()
 {
     throw std::runtime_error(
-            "quorum dkgstatus (detailed)\n"
+            "quorum dkgstatus (proTxHash) (detail_level)\n"
```

Comment:
```suggestion
            "quorum dkgstatus ( "proTxHash" detail_level )\n"
```

Reply:
Not at all, that's a common way of specifying optional params. Look at wallet rpc like `sendtoaddress` or `listtransactions` for example.